### DISCUSSION TITLE
Problem adding https://github.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant in the Custom Repository

### DISCUSSION THREAD

**Discussion by LionelVvv** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#41] [General]
Hi everyone,

HomeAssistant does not accept https://github.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant as a valid add-on repository. 

My HomeAssistant configuration :

- Core 2024.5.1
- Supervisor 2024.04.4
- Operating System 12.2
- Interface utilisateur 20240501.0

Is the url correct ? 

Thanks in advance for any help

Lionel

**Comment by mawoka-myblock** [comment]
The url is correct, but that's about everything I can say. You have to provide some steps on what you did so that I can help troubleshooting.

**Reply by LionelVvv** [reply]
When I try to add a newer repository in Home Assistant I get the following error message : 

![image](https://github.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant/assets/168971867/443fda2e-842b-4ce6-a4c5-e5f155c22fad)

**Reply by mawoka-myblock** [reply]
You have to go to HACS -> Integrations -> Add Custom Repository

You're at Add-Ons -> Custom Repository. This is wrong as you want to add an integration, not an add-on. Please try the above, as the above one is correct. I didn't even know one could add a custom add-on.

**Reply by LionelVvv** [reply]
Thank you for your answer. As I'm new to Home Assistant I was not aware of HACS (I thought it was an acronyme for  "Home Assistant Core System"...) and the way to install it. I think I'm on the right way now ! Not sure yet to reach the destination but i'll try ;-)

**Reply by LionelVvv** [reply]
Everything looks ok now (even if I encoutered the HACS "Waiting for device", but #https://community.home-assistant.io/t/hacs-stuck-waiting-for-device-activation/422010/25 save me) !

Thanks again for the work done to integrate EZ1-M to Home Assistant.

**Reply by mawoka-myblock** [reply]
Sure! Glad I could help!
